client/script.js

```diff
@@ -9,7 +9,8 @@ const descArr = Array.from(desc);
 const tagsArr = Array.from(tags);
 
 const getAll = async () => {
-  const response = await fetch("http://localhost:3500/search");
+  const response = await fetch("http://localhost:5500/search");
+
   const data = await response.json();
 
   for (let i = 0; i < urlArr.length; i++) {
```

Comment:
Could use the createElement function to create new elements, instead of having them already made in the HTML file